van-tracker.ino

```diff
@@ -124,7 +124,7 @@ Adafruit_FONA fona = Adafruit_FONA(99);
 #define USEPLAINSMS_BOOL_1                132
 #define POWERON_BOOL_1                    133
 
-const char STR_HOME[] PROGMEM = " feet\\\\nHome: google.com/search?q=";
+const char STR_HOME[] PROGMEM = " meters\\\\nHome: maps.google.com/?ll=";
```

Comment:
still uncomfortable with this.  STR_HOME is used in
checkLockdownStatus()
handleLockReq()
handleUnlockReq()
handleGeofenceReq()

I would check the math, but to be safe I'd probably increase the size of "message" in each of those functions by 2, or change "meters" to "m"